app/src/main/java/com/onboarding/parkingsystemjava/utils/DateUtils.java

```diff
@@ -0,0 +1,7 @@
+package com.onboarding.parkingsystemjava.utils;
+
+public class DateUtils {
```

Comment:
you can put this constants in the ConstantUtils class

Reply:
done